.pre-commit-config.yaml

```diff
@@ -0,0 +1,36 @@
+
+repos:
```

Comment:
I've read up about pre-commit-config files and these all look OK but I'm not an expert on this subject.